tools/docker-images/clp-package/Dockerfile

```diff
@@ -42,3 +42,4 @@ COPY --link --chown=${UID} ./build/spider/spider-build/src/spider/spider_worker
 COPY --link --chown=${UID} ./build/nodejs-22/bin/node bin/node-22
 COPY --link --chown=${UID} ./build/python-libs/ lib/python3/site-packages/
 COPY --link --chown=${UID} ./build/webui/ var/www/webui/
+COPY --link --chown=${UID} ./build/rust/release/api_server bin/
```

Comment:
(i know i'm the original author of this source file.. sorry for missing that previously)

now that this has become a long list, shall we alphabetize the `COPY`s? i believe the effect will be the same because we have specified `--link`